src/Controller/AppController.php

```diff
@@ -43,6 +43,10 @@ public function initialize()
 
         $this->loadComponent('RequestHandler');
         $this->loadComponent('Flash');
+        //Recommended security enhancements for your application
```

Comment:
Don't agree, neither does this https://github.com/cakephp/app/blob/master/config/bootstrap.php#L168-L170

I'd say `/* foo */` is for inline comments whereas `//$code` would be for out-commented lines.